src/main/app/claims/models/claim.ts

```diff
@@ -91,9 +91,7 @@ export class Claim {
     return !this.countyCourtJudgmentRequestedAt
       && (this.admissionPayImmediatelyPastPaymentDate
         || this.hasDefendantNotSignedSettlementAgreementInTime()
-        || (!this.respondedAt && isPastDeadline(MomentFactory.currentDateTime(), this.responseDeadline)
-        || this.isSettlementAgreementRejected()
```

Comment:
The PR says only about adding tests.
This condition (`this.isSettlementAgreementRejected()`) is removed, @andrewwa-kainos I hope it's not accidental.

Reply:
There was an understanding that any errors found would be fixed (hence some of the other changes). This change was more about cleaning up redundancy though.

Reply:
Thanks